integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -214,10 +214,15 @@ def auto_complete_address(request):
         raise Http404(_("Address could not be found"))
 
     address = result.raw.get("address", {})
+
+    city_town = address.get("city")
+    if not city_town:
+        city_town = address.get("town")
+
     return JsonResponse(
         data={
             "postcode": address.get("postcode"),
-            "city": address.get("city"),
+            "city": city_town,
```

Comment:
```suggestion
            "city": address.get("city") or address.get("town"),
```
Could you give an example address where Nominatim returns `town` instead of `city`?

Reply:
Windgäßchen, 86720 Nördlingen, for example.
Bamberg is also attributed as "town".

Reply:
Interesting, thanks!